agent/globals/globals.go

```diff
@@ -119,6 +119,7 @@ const (
 	AWSSessionToken = "AWS_SESSION_TOKEN"
 	// AWSRoleExpiration is the expansion name for the expiration of a temporary AWS access key.
 	AWSRoleExpiration = "AWS_ROLE_EXPIRATION"
+	HostSecret        = "HOST_SECRET"
```

Comment:
Can we clarify that unlike the other consts here, this isn't actually an expansion, but is instead an env var in the agent?